src/Symfony/Bundle/WebProfilerBundle/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ CHANGELOG
 -----
 
  * added information about orphaned events
+ * added ability to auto-refresh the toolbar if a `Symfony-Debug-Toolbar-Replace` header
+    is set and has a value of '1'
```

Comment:
extra whitespace at the beginning of the line

Reply:
here is what I propose here:
made the toolbar auto-update with info from ajax reponses when they set the `Symfony-Debug-Toolbar-Replace` header to `1`

I would have done it myself, but you disallowed me from pushing on your fork :)